services-geocoding/src/test/resources/forward_intersection.json

```diff
@@ -0,0 +1 @@
tes"}]},{"id":"address.2168135640082828","type":"Feature","place_type":["address"],"relevance":1,"properties":{"accuracy":"intersection"},"text":"Fremont Street","place_name":"Market Street and Fremont Street, Erie, Pennsylvania 16510, United States","center":[-80.0112744,42.139934],"geometry":{"type":"Point","coordinates":[-80.0112744,42.139934]},"context":[{"id":"neighborhood.268738","text":"Brookside"},{"id":"postcode.7318118112165310","text":"16510"},{"id":"place.10004124579967850","wikidata":"Q234061","text":"Erie"},{"id":"region.2157738952111630","short_code":"US-PA","wikidata":"Q1400","text":"Pennsylvania"},{"id":"country.9053006287256050","short_code":"us","wikidata":"Q30","text":"United States"}]},{"id":"address.4254940365412868","type":"Feature","place_type":["address"],"relevance":0.988636,"properties":{"accuracy":"intersection"},"text":"Tremont Street","place_name":"Market Street and Tremont Street, Lynn, Massachusetts 01901, United States","center":[-70.9496726,42.4633684],"geometry":{"type":"Point","coordinates":[-70.9496726,42.4633684]},"context":[{"id":"neighborhood.287332","text":"Central Square Area"},{"id":"postcode.8079700663915320","text":"01901"},{"id":"place.5083986385572820","wikidata":"Q49188","text":"Lynn"},{"id":"region.6776276020561540","short_code":"US-MA","wikidata":"Q771","text":"Massachusetts"},{"id":"country.9053006287256050","short_code":"us","wikidata":"Q30","text":"United States"}]},{"id":"address.7308702273993364","type":"Feature","place_type":["address"],"relevance":0.988636,"properties":{"accuracy":"intersection"},"text":"Tremont Street","place_name":"Market Street and Tremont Street, Presidio, Texas 79845, United States","center":[-104.3747124,29.5627643],"geometry":{"type":"Point","coordinates":[-104.3747124,29.5627643]},"context":[{"id":"postcode.254811532177250","text":"79845"},{"id":"place.8840105903998060","wikidata":"Q970777","text":"Presidio"},{"id":"region.5362387429342410","short_code":"US-TX","wikidata":"Q1439","text":"Texas"},{"id":"country.9053006287256050","short_code":"us","wikidata":"Q30","text":"United States"}]},{"id":"address.1764530839718556","type":"Feature","place_type":["address"],"relevance":0.4016,"properties":{"accuracy":"street"},"text":"2nd & Fremont St","place_name":"2nd & Fremont St, Royal, Nebraska 68773, United States","center":[-98.1238105,42.3344656],"geometry":{"type":"Point","coordinates":[-98.1238105,42.3344656]},"context":[{"id":"postcode.11302267114803110","text":"68773"},{"id":"place.13924027642746770","text":"Royal"},{"id":"region.8407900583987740","short_code":"US-NE","wikidata":"Q1553","text":"Nebraska"},{"id":"country.9053006287256050","short_code":"us","wikidata":"Q30","text":"United States"}]}],"attribution":"NOTICE: © 2019 Mapbox and its suppliers. All rights reserved. Use of this data is subject to the Mapbox Terms of Service (https://www.mapbox.com/about/maps/). This response and the information it contains may not be retained. POI(s) provided by Foursquare."}
```

Comment:
Switch to easier-to-read layout of this JSON response? Use something like https://jsonformatter.curiousconcept.com to get the cleaned up JSON?

Reply:
Sure, will do it, Android Studio also has built in formatter for JSON file :)